scripts/exregional_process_bufr.sh

```diff
@@ -0,0 +1,417 @@
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#
+# Link to the observation lightning bufr files
+#
+#-----------------------------------------------------------------------
+
+run_lightning=false
+obs_file=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.lghtng.tm00.bufr_d
+print_info_msg "$VERBOSE" "obsfile is $obs_file"
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "lghtngbufr"
+   run_lightning=true
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable for lightning
+#
+#-----------------------------------------------------------------------
+
+cat << EOF > lightning_bufr.namelist
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  minute=00,
+  trange_start=-10,
+  trange_end=10,
+  bkversion=1,
+ /
+
+EOF
+
+#
+#-----------------------------------------------------------------------
+#
+# link/copy observation files to working directory 
+#
+#-----------------------------------------------------------------------
+#
+EXEC="${EXECDIR}/process_Lightning_bufr.exe"
+
+if [ -f $EXEC ]; then
+  print_info_msg "$VERBOSE" "
+Copying the lightning process  executable to the run directory..."
+  cp_vrfy ${EXEC} ${WORKDIR}/process_Lightning_bufr.exe
+else
+  print_err_msg_exit "\
+The executable specified in EXEC does not exist:
+  EXEC = \"$EXEC\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process for lightning bufr file 
+#
+#-----------------------------------------------------------------------
+#
+if [ $run_lightning ]; then
+   $APRUN ./process_Lightning_bufr.exe > stdout_lightning_bufr 2>&1 || print_err_msg "\
+   Call to executable to run lightning process returned with nonzero exit code."
+fi
+
+
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#
+# Link to the observation NASA LaRC cloud bufr file
+#
+#-----------------------------------------------------------------------
+
+obs_file=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.lgycld.tm00.bufr_d
+print_info_msg "$VERBOSE" "obsfile is $obs_file"
+run_cloud=false
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "NASA_LaRC_cloud.bufr"
+   run_cloud=true
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable for NASA LaRC cloud
+#
+#-----------------------------------------------------------------------
+
+cat << EOF > namelist_nasalarc
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  bufrfile='NASALaRCCloudInGSI_bufr.bufr',
```

Comment:
This filename does not match the one copied into the directory. Does that matter?

Reply:
doesn't matter. This one is not used by RRFS. But this line needs to be here.

Reply:
Are these namelists documented somewhere? It might be nice to add where they're documented here to the script since most of us aren't familiar with them.

Reply:
Added description for all namelist parameters generated in the scripts.